src/validataclass/validators/dataclass_validator.py

```diff
@@ -30,15 +30,6 @@
 # Type variable for an instance of a dataclass
 T_Dataclass = TypeVar('T_Dataclass', bound=object)
 
-# Define type alias for dataclasses.Field
-# NOTE: In Python >= 3.9, dataclasses.Field is a Generic, so mypy will complain if no type parameter is given.
-# However, Field[Any] will raise a runtime error in Python 3.8 because there the type is not parametrized yet.
-# TODO: Replace type alias with dataclasses.Field[Any] when removing Python 3.9 support. (#15)
```

Comment:
Issue #15 still exists. Is it still needed, or can it be closed?

Reply:
Issue #15 is solved by this PR, so it can be closed as soon as this is merged. ^^